src/Service.Tests/SqlTests/RestApiTests/Patch/MsSqlPatchApiTests.cs

```diff
@@ -313,6 +331,79 @@ public virtual async Task PatchOneWithRowversionFieldInRequestBody()
                 expectedSubStatusCode: DataApiBuilderException.SubStatusCodes.BadRequest.ToString()
                 );
         }
+        /// <summary>
+        /// Test to validate that even when an update DML trigger is enabled on a table, we still return the
+        /// latest data as it is present after the trigger gets executed. Whenever an update DML trigger is enabled,
+        /// we use a subsequent SELECT query to get the data instead of using OUTPUT clause. To validate that the data is returned
+        /// after the trigger is executed, we use the new values (written by the trigger) in the WHERE predicates of the verifying sql query.
+        /// </summary>
+        [TestMethod]
+        public async Task PatchOneUpdateInTableWithUpdateTrigger()
+        {
+            // Validate that PATCH operation (resulting in update) succeeds when an update DML trigger is configured for a table
```

Comment:
Refer to https://github.com/Azure/data-api-builder/pull/1630#discussion_r1325210147